package.json

```diff
@@ -2,28 +2,29 @@
   "name": "pokemon-showdown",
   "description": "The server for the Pokémon Showdown battle simulator",
   "version": "0.11.7",
-  "main": ".sim-dist/index.js",
+  "main": "sim/index.ts",
   "dependencies": {
+    "@swc/core": "^1.2.61",
     "@types/better-sqlite3": "^5.4.0",
     "probe-image-size": "^5.0.0",
     "sockjs": "^0.3.21",
-    "sucrase": "^3.15.0"
+    "ts-node": "^10.0.0"
   },
   "optionalDependencies": {
-    "better-sqlite3": "^7.1.0",
     "brain.js": "^2.0.0-beta.2",
     "cloud-env": "^0.2.3",
     "node-static": "^0.7.11",
     "nodemailer": "^6.4.6",
     "permessage-deflate": "^0.1.7",
     "sql-template-strings": "^2.2.2",
-    "sqlite": "^3.0.6"
+    "sqlite": "^3.0.6",
+    "sucrase": "^3.15.0"
```

Comment:
?

Reply:
Sucrase is an optional dependency now because it's only needed to build the NPM package.